ProjectLighthouse/Serialization/LbpOutputFormatter.cs

```diff
@@ -26,12 +26,18 @@ protected override bool CanWriteType(Type type)
         if (isSerializable) return base.CanWriteType(type);
         Logger.Warn($"Unable to serialize type '{type?.Name}' because it doesn't extend ISerializable: (fullType={type?.FullName}", LogArea.Serialization);
         return false;
-    } 
+    }
 
     public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
     {
         if (context.Object is not ILbpSerializable o) return;
 
+        if (o is LbpCustomXml manualXml)
```

Comment:
```suggestion
        if (o is LbpCustomXml customXml)
```
ik this is from when the class was called ManualXml so lets just change this to match the new name